redish/libs/redish-frontend/authentication/data-access/src/lib/connector/authentication-api.service.ts

```diff
@@ -1,27 +1,49 @@
-import { CreateUserDTO } from '@redish-shared/domain';
+import {
+  AuthenticateUserDto,
+  CreateUserDto,
+  RedishError,
+  TokenDto,
+  UuidDto,
+} from '@redish-shared/domain';
 import axios from 'axios';
 
-export class AuthenticationService {
-  private baseURL = 'https://localhost:3000/auth';
+export class AuthenticationApiService {
+  // todo: move to config
+  private baseURL = 'http://localhost:3000/user';
 
   public async register(
     username: string,
     email: string,
     password: string
   ): Promise<string | null> {
-    const url = `${this.baseURL}/login`;
-    const data: CreateUserDTO = { username, password, email };
+    const url = `${this.baseURL}/register`;
+    const data: CreateUserDto = { username, password, email };
     try {
-      const uuid = await axios
-        .post<string>(url, data)
-        .then((response) => response.data);
-      return uuid;
+      const response = await axios.post<UuidDto>(url, data);
+      return response.data.uuid;
+      // todo: redish error?
     } catch (error) {
       console.error(error);
```

Comment:
Will the error handling occur in this service or will it be handled by the class that calls the service? It's usually better to handle the error earlier